deploy/examples/operator.yaml

```diff
@@ -449,6 +449,14 @@ data:
   # ROOK_CSIADDONS_IMAGE: "quay.io/csiaddons/k8s-sidecar:v0.5.0"
   # The CSI GRPC timeout value (in seconds). It should be >= 120. If this variable is not set or is an invalid value, it's default to 150.
   CSI_GRPC_TIMEOUT_SECONDS: "150"
+
+  # Enable topology based provisioning.
+  CSI_ENABLE_TOPOLOGY: "false"
+  # Domain labels define which node labels to use as domains
+  # for CSI nodeplugins to advertise their domains
+  # NOTE: the value here serves as an example and needs to be
+  # updated with node labels that define domains of interest
+  # CSI_TOPOLOGY_DOMAIN_LABELS: "topology.kubernetes.io/zone,topology.kubernetes.io/hostname,topology.rook.io/rack"
```

Comment:
Same as above 
hostname should be "kubernetes.io/hostname",

lets keep
`CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname, topology.kubernetes.io/region, topology.kubernetes.io/zone, topology.rook.io/rack"`